data-transformers/transformers/statsTransformer.py

```diff
@@ -97,9 +127,46 @@ def createIODict(a, trialID, experimentID, containerID, hostID):
         dicts.append(d)
     return dicts
 
-def getFromMinio(url):
-    from commons import getFromUrl
-    return getFromUrl(url)
+def getBytesSum(e, PIDS):
+    e = filter((lambda a: not "Refreshing" in a), e)
+    e = map((lambda a: a.strip("\n").split("\t")), e)
+    e = filter((lambda a: len(a) == 3 and a[0].split("/")[-2] in PIDS), e)
+    if len(e) == 0:
+        return (0, 0)
+    e = map((lambda a: (float(a[1])*1000,float(a[2])*1000)), e)
+    e = reduce((lambda a, b: (a[0]+b[0],a[1]+b[1])), e)
+    return e
+
+def createNetworkHostQuery(sc, top, net, trialID, experimentID, containerID, hostID):
+    def getPid(a):
+        data = json.loads(a.decode())
+        pids = []
+        for p in data["Processes"]:
+            pids.append(p[2])
+        return pids
+    PIDS = sc.parallelize(top).map(getPid).reduce(lambda a, b: a+b)
+    
+    netPerSecond = []
+    chunk = []
+    for line in net[1:]:
+        if "Refreshing" in line:
+            netPerSecond.append(chunk)
+            chunk = []
+        else:
+            chunk.append(line)
+
+    data = sc.parallelize(netPerSecond).map(lambda a: getBytesSum(a, PIDS)).collect()
+    i = 0
```

Comment:
Basically, for every block (starts with "Refreshing" and lists a bunch of processes doing stuff), it sums the received and sent bytes of all of them, but skipping those who are not in PIDS, meaning we ignore those not from the observed container.